Dockerfile.builder

```diff
@@ -0,0 +1,28 @@
+# MIT License
+#
+# (C) Copyright [2022] Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# Docker file to build a temporary builder image so we can control the
+# build environment for switchboard when building the RPMs.
+
+FROM arti.dev.cray.com/baseos-docker-master-local/sles15sp2:latest as base
```

Comment:
Should we use `artifactory.algol60.net/registry.suse.com/suse/sle15:15.3` as the base instead of a container in arti?

Reply:
That seems reasonable.  I haven't tried that, but I would love to get that switched over. I will give it a try, and, if it. works, go with it.  If that works there I will probably switch the Broker UAI over to using that too.  I didn't know where to find that in Algol60, which is why I didn't do it originally.